### PR TITLE
PR: API endpoints for automation and integration (#910)

### PR DESCRIPTION
**Pull request by goekay**
Feature request: https://github.com/steve-community/steve/issues/910

### CODE REVIEW

**Comment by kohlerdominik** [comment]
Up to you regarding my suggested doc change. 

But other than that, we are excited to see this rolled out. Thanks very much for your work!

**Review by kohlerdominik — APPROVED** [review]

File: src/main/java/de/rwth/idsg/steve/web/dto/QueryForm.java [hunk 1]
```diff
@@ -36,25 +37,35 @@
 @ToString
 public abstract class QueryForm {
 
+    @ApiModelProperty(value = "The identifier of the chargebox (i.e. charging station)")
     private String chargeBoxId;
+
+    @ApiModelProperty(value = "The OCPP tag")
     private String ocppIdTag;
 
+    @ApiModelProperty(value = "The start of the query time period. A date/time without timezone. Example: 2022-10-10T09:00")
```
[the remark this answers is not in the document]
Comment (goekay) [reply src/main/java/de/rwth/idsg/steve/web/dto/QueryForm.java]: i already have a commit that is quite similar to this suggestion: [9ad5abf](https://github.com/steve-community/steve/pull/987/commits/9ad5abf48d949fe8edfced44890f882c6cff2c7c)

File: src/main/java/de/rwth/idsg/steve/web/dto/QueryForm.java [hunk 2]
```diff
@@ -36,25 +37,35 @@
...
     private LocalDateTime from;
+
+    @ApiModelProperty(value = "The end of the query time period. A date/time without timezone. Example: 2022-10-10T12:00")
```
[the remark this answers is not in the document]
Comment (goekay) [reply src/main/java/de/rwth/idsg/steve/web/dto/QueryForm.java]: i already have a commit that is quite similar to this suggestion: [9ad5abf](https://github.com/steve-community/steve/pull/987/commits/9ad5abf48d949fe8edfced44890f882c6cff2c7c)

**Comment by kohlerdominik** [comment]
perfect :)

**Pull request merged**